vermouth/__init__.py

```diff
@@ -37,9 +37,9 @@
     import atexit
     from contextlib import ExitStack
 except ImportError:
-    import os
-    DATA_PATH = os.path.join(os.path.dirname(__file__), 'data')
-    del os
+    from pathlib import Path
+    DATA_PATH = Path(__file__).parent / 'data'
```

Comment:
Pathlib works os independent right?

Reply:
Yes, that's kind of the whole point. `os.path` does as well, but that API is less nice since you need to do `os.path.join` all the time rather than inlining a `/`